Applications/WebPortal/WebApp/library/source/pages/account_page/compliance_page/compliance_page.tsx

```diff
@@ -11,22 +12,29 @@ interface Properties {
   /** The set of available currencies to select. */
   currencyDatabase?: Nexus.CurrencyDatabase;
 
-  /** The list of compliance rules. */
+  /** The list of compliance rules to display and edit. */
   complianceList: Nexus.ComplianceRuleEntry[];
```

Comment:
Rename to entries. Generally for naming conventions the default name is the name of the type with the context inferred. So you can rename complianceList to ```entries```, and ```ruleSchemas``` to ```schemas```. If there were more than one kind of entry, like an XXX entry and a YYY entry, then you would name the variable ```xxxEntry``` and ```yyyEntry``` to differentiate the two.